docker/Dockerfile

```diff
@@ -25,10 +25,10 @@ ENV BOOST_HOME=/usr/include/boost
 ARG CPPFLAGS="-I${BOOST_HOME} -I${BOOST_HOME}/boost"
 ARG GT4PY_OPTIONALS=""
 COPY external/gt4py /gt4py
+RUN git clone --depth 1 -b v2.1.0 https://github.com/GridTools/gridtools.git /gt4py/src/gt4py/_external_src/gridtools2
 
 RUN if [ ! -z `echo $GT4PY_OPTIONALS | grep cuda` ] ; then pip install cupy-cuda102==7.7.0 ; else echo Not installing cuda ; fi
-RUN pip install --no-cache-dir -c /constraints.txt "/gt4py${GT4PY_OPTIONALS}" && \
-    python3 -m gt4py.gt_src_manager install
+RUN python -m pip install --no-cache-dir -c /constraints.txt -e "/gt4py${GT4PY_OPTIONALS}"
```

Comment:
When installing without -e, on the first gtc backend run, it pulls the wrong version of gridtools again. But having -e skips that step. Unclear about this behavior.

Reply:
`-e` is directing the GT source to be install alongside the python source, while without the `-e` PIP installs into its own directory structure. The issue is that by default the source path is set to the one created by -e.